### DISCUSSION TITLE
Buff les épées doubles

### DISCUSSION THREAD

**Discussion by BananePlantain** [Crownicles/Crownicles#1924] [Suggestions fermées]
Je trouve que les épées doubles (104 attaque,30 défense et -50 vitesse,arme spécial) ont une débuff trop important de vitesse par rapport à la rareté de l'arme et de sa puissance notamment par rapport à la bombe atomique (239 attaque et -40 vitesse,arme mythique).
Qu'en pensez vous ?

**Comment by OrionYp** [comment]
Les épées double font partie des armes spéciales, avec 104 d’attaque 30 de défense et -50 de vitesse le bonus reste de 104+30-50=84.
La pire arme de cette catégorie fait 48 d’attaque sans bonus ou malus, il y en a aussi une a 52 d’attaque sans bonus.
Les épées double sont suffisamment puissantes pour mériter un nerf de 50 de vitesse d’après moi

**Reply by BananePlantain** [reply]
c'est puissant mais 50 c'est un gros débuff

**Comment by OrionYp** [comment]
Cette arme est surtout utile aux classes qui n’ont pas besoin de vitesse (tank/paladin et classes correspondantes) elle a pour but de les aider sans être trop rare, il existe l’arc de même rareté si tu préfères la vitesse

**Reply by BananePlantain** [reply]
mouais pas faux
